src/main/java/mate/controller/GetAllManufacturersController.java

```diff
@@ -0,0 +1,25 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/manufacturers")
+public class GetAllManufacturersController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final ManufacturerService manufacturerService
+            = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.setAttribute("manufacturers", manufacturerService.getAll());
+        req.getRequestDispatcher("/WEB-INF/manufacturers/allManufacturers.jsp")
```

Comment:
```suggestion
        req.getRequestDispatcher("/WEB-INF/manufacturers/all.jsp")
```
Please fix the same issue in all places